Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,409 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        private const string currencySymbol = "¤";
+        private const string invalidMessage = "N/A.";
+        private const string messageEnd = ".";
+        private static readonly NumberFormatInfo numberFormatInfo = new NumberFormatInfo { NumberDecimalSeparator = messageEnd };
+
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (!ArrayIsValid(peopleAndBalances)) return invalidMessage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4970157)